content/news/this-week-in-fluvio-0001.md

```diff
@@ -0,0 +1,185 @@
+---
+title: This Week in Fluvio 1
```

Comment:
Should we put here a date?

Reply:
Hmm probably, I wonder if there is a default rendering if we give one

Reply:
Added the field to metadata but I don't think it gets rendered. We can fix in the future